classes/fakeroot.oeclass

```diff
@@ -3,7 +3,7 @@
 ## By inheriting this class, it is ensures that the native version of fakeroot
 ## build, is used instead of fakeroot from the build system.
 
-CLASS_DEPENDS += "native:fakeroot"
+CLASS_DEPENDS += "native:fakeroot native:libacl1-dev"
```

Comment:
You are right, I have moved the dependency from the class to the recipe.

Building fakeroot with libacl1-dev *not* installed:
```
checking sys/acl.h usability... no
checking sys/acl.h presence... no
checking for sys/acl.h... no
```

Building fakeroot with libacl1-dev installed:
```
checking sys/acl.h usability... yes
checking sys/acl.h presence... yes
checking for sys/acl.h... yes
```